README.md

```diff
@@ -1,7 +1,6 @@
-EOS.IO API library for Go
```

Comment:
Revert those changes, I guess they are related to your own fork of the library.

Reply:
I've reverted, sry